src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsOnScoreView.qml

```diff
@@ -60,9 +62,7 @@ Item {
         instrumentsView.positionViewAtEnd()
     }
 
-    InstrumentsOnScoreListModel {
-        id: instrumentsOnScoreModel
-    }
+
 
```

Comment:
Make sure to delete unnecessary whitespace; one empty line is enough :)

Reply:
Ok, got it